crates/bevy_pbr/src/wireframe.rs

```diff
@@ -38,138 +30,97 @@ impl Plugin for WireframePlugin {
 
         app.register_type::<WireframeConfig>()
             .init_resource::<WireframeConfig>()
-            .add_plugin(ExtractResourcePlugin::<WireframeConfig>::default());
-
-        if let Ok(render_app) = app.get_sub_app_mut(RenderApp) {
-            render_app
-                .add_render_command::<Opaque3d, DrawWireframes>()
-                .init_resource::<WireframePipeline>()
-                .init_resource::<SpecializedMeshPipelines<WireframePipeline>>()
-                .add_system_to_stage(RenderStage::Extract, extract_wireframes)
-                .add_system_to_stage(RenderStage::Queue, queue_wireframes);
-        }
-    }
-}
-
-fn extract_wireframes(mut commands: Commands, query: Extract<Query<Entity, With<Wireframe>>>) {
-    for entity in query.iter() {
-        commands.get_or_spawn(entity).insert(Wireframe);
+            .add_plugin(MaterialPlugin::<WireframeMaterial>::default())
+            .add_system(apply_global)
+            .add_system(apply_material);
     }
 }
 
-/// Controls whether an entity should rendered in wireframe-mode if the [`WireframePlugin`] is enabled
+/// Toggles wireframe rendering for any entity it is attached to.
+///
+/// This requires the [`WireframePlugin`] to be enabled.
 #[derive(Component, Debug, Clone, Default, Reflect)]
 #[reflect(Component, Default)]
 pub struct Wireframe;
 
+/// Configuration resource for [`WireframePlugin`].
 #[derive(Debug, Clone, Default, ExtractResource, Reflect)]
 #[reflect(Resource)]
 pub struct WireframeConfig {
-    /// Whether to show wireframes for all meshes. If `false`, only meshes with a [Wireframe] component will be rendered.
+    /// Whether to show wireframes for all meshes.
+    /// If `false`, only meshes with a [`Wireframe`] component will be rendered.
     pub global: bool,
 }
 
-pub struct WireframePipeline {
```

Comment:
This public struct was removed, hence the breaking change label.

Reply:
Oh, right, although, I'm not sure why this was pub in the first place.